Tests/SwiftStdlibTests/DoubleExtensionsTests.swift

```diff
@@ -91,5 +91,29 @@ final class DoubleExtensionsTests: XCTestCase {
 		}
 		XCTAssert(num.asLocaleCurrency.contains("\(num)"))
 	}
+    
+    func testUnitsOfMeasurement() {
```

Comment:
We have to split this up 😕 See #389. My fault for not including this in CONTRIBUTING.md